main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/AlexeyVilkov/go_final_project/database"
+	"github.com/AlexeyVilkov/go_final_project/handlers"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+func main() {
+
+	// подключаемся к БД
+	_, err := database.ConnectDB()
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	// отключаемся по завершении функции
+	defer database.Db.Close()
+
+	// устанавливаем значение переменной окружения TODO_PORT для проверки
+	//os.Setenv("TODO_PORT", "8080")
+
+	port := ":7540" //значение по умолчанию, из tests: strconv.Itoa(tests.Port)
+
+	if value, exists := os.LookupEnv("TODO_PORT"); exists {
+		port = ":" + value
+	}
+
+	r := http.FileServer(http.Dir("web"))
+	http.Handle("/", r)
+	http.HandleFunc("/api/nextdate", handlers.GetNextDate)
+	http.HandleFunc("/api/task", handlers.ActionTask)
+	http.HandleFunc("/api/tasks", handlers.GetTasks)
+	http.HandleFunc("/api/task/done", handlers.DoneTask)
+
+	fmt.Println("Server is listening...")
```

Comment:
Можно сразу порт написать, что было проще ориентироваться
```suggestion
	fmt.Println("Server is listening...", port)
```